src/fnet/fnet_model.rs

```diff
@@ -958,7 +965,7 @@ impl FNetForQuestionAnswering {
     /// # let device = Device::Cpu;
     /// # let vs = nn::VarStore::new(device);
     /// # let config = FNetConfig::from_file(config_path);
-    /// let model = FNetForTokenClassification::new(&vs.root(), &config);
+    /// let model = FNetForTokenClassification::new(&vs.root(), &config).unwrap();
```

Comment:
it looks like this docstring was wrong to start with - could you please fix it with the question answering model instead?

Reply:
Done :)